utils/compare/graph.py

```diff
@@ -0,0 +1,306 @@
+import io
+import pprint
+from json import loads
+import pandas as pd
+from search import ElasticService
+import numpy as np
+
+def graph(oMetrics, nMetrics):
+    metrics = []
+    x=[]
+    y=[]
+    for index, row in oMetrics.iterrows():
+        test = "{}-{}".format(row['profile'], row['messageSize'])
+        value = "{}".format(row['throughput'])
+        x.append(value)
+        y.append(test)
+    old = {'y' : x,
+        'x' : y,
+        'name' : 'Previous results average',
+        'type' : 'bar',
+        'orientation' : 'v'}
+    x=[]
+    y=[]
+    for index, row in nMetrics.iterrows():
+        test = "{}-{}".format(row['profile'], row['messageSize'])
+        value = "{}".format(row['throughput'])
+        x.append(value)
+        y.append(test)
+    new = {'y' : x,
+        'x' : y,
+        'name' : 'Current results average',
+        'type' : 'bar',
+        'orientation' : 'v'}
+    metrics.append(old)
+    metrics.append(new)
+
+    return metrics
+
+
+def get_final_graph_metrics(metric_of_interest, divider,previous_data, current_data): 
```

Comment:
What is the idea behind these plotly graph functions? I couldn't see any invocations of them in this PR.